cm_api.py

```diff
@@ -605,6 +636,17 @@ def _get(self, endpoint, params=None):
         response.raise_for_status()
         return response
 
+    def _get_with_relative_url(self, url_suffix, params=None):
```

Comment:
This doesn't seem like a good solution. I'd rather update the `_get` method and change it so that there's a keyword arg there called `endpoint_suffix` that defaults to `api`, but that can be changed. A whole new method seems overkill.

Reply:
Done.